docs/references/config/genesis.json.md

```diff
@@ -0,0 +1,211 @@
+  "chain_id": "test-chain-HfdKnD",
+  "initial_height": "0",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "4194304",
+      "max_gas": "10000000"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "1048576"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {
+      "app": "0"
+    },
+    "abci": {
+      "vote_extensions_enable_height": "0"
+    }
+  },
+  "validators": [
+    {
+      "address": "E74FBE24164CFC4F88E311C3AC92E63D0DC310D8",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "UjxDQgVTlHJOZ7axpMl/iczMIJXiQpFxCFjwKGvzYqE="
+      },
+      "power": "10",
+      "name": ""
+    }
+  ],
+  "app_hash": ""
+}
+```
+
+For a production example, you can see [here](https://github.com/cosmos/mainnet/tree/master/genesis)
+the history of genesis files for the Cosmos Hub network.
+
+## genesis_time
+Timestamp of the genesis file creation.
+
+| Value type          | string                     |
+|:--------------------|:---------------------------|
+| **Possible values** | RFC3339-formatted datetime |
+
+RFC3339 has multiple representation. The one we use here has 6 digits for sub-second representation.
+
+## chain_id
+The chain ID of the blockchain network.
+
+| Value type          | string                            |
+|:--------------------|:----------------------------------|
+| **Possible values** | usually in `"name-number"` format |
+
+Cannot be empty.
+
+Can be maximum 50 UTF-8-encoded character.
+
+The `number` part is typically a revision number of the blockchain, starting at `1` and incrementing each time the network
+undergoes a significant upgrade.
+
+## initial_height
+Initial height at genesis.
+
+| Value type          | string      |
+|:--------------------|:------------|
+| **Possible values** | &gt;= `"0"` |
+
+When a hard fork happens, a new chain can start from a higher initial height by setting this parameter.
+
+Height `"0"` is the same as height `"1"` and it means the first ever block in the chain.
+
+> Note: A height in CometBFT is an `int64` integer therefore its maximum value is `9223372036854775807`
+
+## consensus_params.block.max_bytes
+Maximum block size in bytes.
+
+| Value type          | string     |
+|:--------------------|:-----------|
+| **Possible values** | &gt; `"0"` |
+| **Possible values** | `"-1"`     |
+
+`"-1"` means the hard-wired maximum of 104857600 bytes.
+
+This parameter cannot be `0`.
+
+## consensus_params.block.max_gas
+Maximum gas allowed.
+
+| Value type          | string      |
+|:--------------------|:------------|
+| **Possible values** | &gt;= `"0"` |
+| **Possible values** | `"-1"`      |
+
+`"-1"` means unlimited gas.
+
+## consensus_params.evidence.max_age_num_blocks
+Limit evidence of malfeasance based on how old the evidence is. This parameter limits the evidence based on its block
```

Comment:
We usually use `misbehaviour` instead of `malfeasance`
```suggestion
Limit evidence of misbehaviour based on how old the evidence is. This parameter limits the evidence based on its block
```